### PR TITLE
tests: add case to ensure entry and output resolve path same way

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
Tests.

Refers #899 

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
NA

**Summary**

this ensures `entry` and `output` resolve the path the same way

**Does this PR introduce a breaking change?**

NO

**Other information**

### CODE REVIEW

**Comment by snitin315** [comment]
Friendly ping @rishabh3112

File: test/entry/flag-entry/entry-with-flag.test.js [hunk 1]
```diff
@@ -22,6 +22,23 @@ describe('entry flag', () => {
         });
     });
 
+    it('should resolve path src/a.js to ./src/a.js', (done) => {
```
Comment (jamesgeorge007) [inline test/entry/flag-entry/entry-with-flag.test.js:25]:
```suggestion
    it('should resolve the path to src/a.js as ./src/a.js', (done) => {
```

File: test/output/named-bundles/output-named-bundles.test.js [hunk 2]
```diff
@@ -23,6 +23,14 @@ describe('output flag named bundles', () => {
         expect(stats.isFile()).toBe(true);
     });
 
+    it('should resolve the path binary/a.bundle.js as ./binary/a.bundle.js', () => {
```
Comment (jamesgeorge007) [inline test/output/named-bundles/output-named-bundles.test.js:26]:
```suggestion
    it('should resolve the path to binary/a.bundle.js as ./binary/a.bundle.js', () => {
```

**Pull request merged**